doc/samtools-idxstats.1

```diff
@@ -61,6 +61,14 @@ name, sequence length, # mapped read-segments and # unmapped
 read-segments. It is written to stdout.  Note this may count reads
 multiple times if they are mapped more than once or in multiple fragments.
 
+.SH OPTIONS
+.TP 11
+.B -X
+This option will allow the user to specify a customised index file
+location. Example usage: samtools idxstat [options] -X </data_folder/data.bam>
+[/index_folder/index.bai]
```

Comment:
I'd be inclined to removed the <> and [] bits in this example for filename and index.  See the EXAMPLES section of samtools-view.1 which has a `-X` option.

If not, then `[`...`]` is misleading as it implies it's optional, but when using `-X` it's mandatory.

Suggestion:

```
This option will allow the user to specify a customised index file
location.  E.g.

.B samtools idxstats [options] -X /data/in.bam /index/in.bai                                           
```

Reply:
I was following the example in samtools-depth.1.

Reply:
That's not using angle brackets though and it doesn't have the mandatory components in square brackets (except where they're optional because the BAM filename was optional too).  Ie it's essentially `fn1 [fn2 ...] idx1 [idx2 ...]`. 

Granted we're not great on consistency though.